lib/cpp/Kconfig

```diff
@@ -107,7 +107,12 @@ config ARCMWDT_LIBCPP
 config EXTERNAL_LIBCPP
 	bool "External C++ standard library"
 	help
-	  Build with an external/user-provided C++ standard library.
+	  Build and link with an external/user-provided C++ standard library.
+
+config EXTERNAL_MODULE_LIBCPP
```

Comment:
Should this be named `MODULE_LIBCPP` in case we end up with a supported Zephyr module for a standard C++ library?

Reply:
Same as previous comment, let me know.